v2/internal/controllers/crd_networking_privateendpoint_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-service-operator/v2/api/network/v1beta20201101"
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1beta20220701"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
+	storage "github.com/Azure/azure-service-operator/v2/api/storage/v1beta20210401"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+)
+
+func Test_Networking_PrivateEndpoint_CRUD(t *testing.T) {
```

Comment:
Did you reference an ARM template/bicep example to produce this test? If so can you link it in a comment up here at the top?

Reply:
There is an ARM bicep example identical to this, however, they use `SQLServer` and we're using `StorageAccount` in this test. Still do we add the link?

Reply:
Yeah I think still worth adding the link. Just say you replaced SQLServer with StorageAccount

Reply:
I don't see the comment, yet.

Reply:
Added!